### PR TITLE
Initial analyzer implementation

### PR DESCRIPTION
**Pull request by bzz**
Part of the https://github.com/src-d/lookout/issues/140

Analyzer in python that uses https://github.com/src-d/lookout-sdk and wraps https://github.com/bblfsh/sonar-checks

# TODOs
 - [x] doc, license
 - [x] requirements.txt
 - [x] initial analyzer draft
 - [x] ENV var flags
 - [x] call `bblfsh_sonar_checks`
 - [x] Docker
 - [x] travis
 - [x] release \w `docker-push`
 - [x] k8s
 - [x] bump `bblfhs_sonar_check` lib to a release with bblfsh/sonar-checks#6

### CODE REVIEW

**Comment by bzz** [comment]
CI will pass as soon as new `bblfhs_sonar_check` lib is release with https://github.com/bblfsh/sonar-checks/pull/6

File: README.md [hunk 1]
```diff
@@ -1 +1,40 @@
-# lookout-sonarcheck-analyzer
\ No newline at end of file
+# [![Build Status](https://travis-ci.org/src-d/lookout-sonarcheck-analyzer.svg)](https://travis-ci.org/src-d/lookout-sonarcheck-analyzer) lookout analyzer: sonarcheck
+
+A [lookout](https://github.com/src-d/lookout/) analyzer implementation that uses bblfsh UAST and a [sonar-checks](https://github.com/bblfsh/sonar-checks).
```
Comment (carlosms) [inline README.md]: s/and a sonar-checks/and sonar-checks/

File: README.md [hunk 2]
```diff
@@ -1 +1,40 @@
...
+
+_Disclamer: this is not official product, but only serves the purpose of testing the lookout._
```
Comment (carlosms) [inline README.md]: proposal to reword this:
this is not an official product, it only serves the purpose of testing lookout.

File: README.md [hunk 3]
```diff
@@ -1 +1,40 @@
...
+
+# Development
+## Release
+
+Main release artifact is a Docker image, so
+ 
+  - `make docker-push`
+
+
+# Licens
```
Comment (carlosms) [inline README.md]: License

**Comment by bzz** [comment]
Review addressed, new version of sonar-check applied.

Should be ready to merge and release 0.0.1 after CI is green.

**Pull request merged**